decorator_allowed_groups/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2021-Today: GRAP (<http://www.grap.coop/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Decorator - Allowed Groups",
```

Comment:
You ix single and double quotes in this file. I would recommend to black the whole module, easing migration to subsequent versions where this has to be done anyway.

Reply:
Thanks ! Done.